### ISSUE TITLE
PhET sims files shared by WhatsApp are not opening in iPhones and some Huawei

### ISSUE THREAD

**Issue by DianaTavares** [phetsims/chipper#1255]
I am not sure if this is an issue that PhET can solve, but a teacher share with me that he download the sim for our website and send the file to the students in a WhatsApp group (WhatsApp is free in Mexico for most the cellphone companies). The problem is that the students with iPhones and some Huawei can't open the sim. They see only a black screen and nothing happen. 
If these students download the sim directly from the PhET website, they can open the file, looks like the problem is when they try to open it by giving a click in the file on  WhatsApp. 
Is there something that we can do?

**Comment by KatieWoe** [comment]
@kathy-phet what do you need for testing? Should we try to recreate it or work with a developer to find the source? If so, who should we work with?

**Comment by oliver-phet** [comment]
> If these students download the sim directly from the PhET website, they can open the file, looks like the problem is when they try to open it by giving a click in the file on  WhatsApp. 

I don't think this has anything to do with WhatsApp. iPhones don't allow the user to open local web pages/html files that are downloaded to the device. That's why we have instructions for adding sims to the browsers reading list: https://phet.colorado.edu/en/help-center/running-sims/tablet

I'd be curious to find out how they are able to download and open the sim downloaded directly from the website.

**Comment by samreid** [comment]
This does not seem like high priority and I haven't heard plans to try to reproduce this problem, so I will mark it as deferred for now.  Anyone feel free to change the prioritization.
